src/app/personside/infotabs/meldinger/traadliste/TraadListeElement.tsx

```diff
@@ -45,7 +45,7 @@ function sisteSendteMelding(traad: Traad) {
 }
 
 function TraadListeElement(props: Props) {
-    const datoTekst = datoVerbose(props.traad.dato).meldingerFormat;
+    const datoTekst = formatDate(props.traad.dato, DATOFORMAT_TRAADER);
```

Comment:
Tenker at vi bør ha en funksjon pr datoformat. Her må man liksom importere både funksjon og formatering. Da er det enklere om formateringen blir abstrahert ut i funksjonen.

Reply:
f.eks `formaterDatoMedKlokkeslett()` og  `formaterDato()`

Reply:
Må si meg enig, er fan av funksjoner med beskrivende navn som dekker for enkle parametre som dette. Spesielt når de blir gjenbrukt så mye.